src/content/en/updates/2020/10/http-cache-partitioning.md

```diff
@@ -0,0 +1,261 @@
+same image (`https://x.example/doge.png`). Though the same image was loaded in
+the previous example, since the key doesn't match it will not be a cache hit.
+
+The image is requested from the network and cached using `https://b.example` +
+`https://b.example` + `https://x.example/doge.png` as the key.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-6.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://a.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+Now the user comes back to `https://a.example` but this time the image
+(`https://x.example/doge.png`) is embedded within an iframe. In this case, the
+key is `https://a.example` + `https://a.example` + `https://x.example/doge.png`
+and a cache hit occurs. (Note that when the top-level site and the iframe are
+the same site, the resource cached with the top-level frame can be used.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-4.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://c.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+The user is back at `https://a.example` but this time the image is hosted in an
+iframe from `https://c.example`.
+
+In this case, the image is downloaded from the network because there is no
+resource in the cache that matches the key `https://a.example` +
+`https://c.example` + `https://x.example/doge.png`.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-7.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://c.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+What if the domain contains a subdomain or a port number? The user visits
+`https://subdomain.a.example`, which embeds an iframe
+(`https://c.example:8080`), which requests the image.
+
+Because the key is created based on "scheme://eTLD+1", subdomains and port
+numbers are ignored, hence a cache hit occurs.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-5.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://a.example</code>, <code>https://c.example</code>, <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+What if the iframe is nested multiple times? The user visits
+`https://a.example`, which embeds an iframe (`https://b.example`), which embeds
+yet another iframe (`https://c.example`), which finally requests the image.
+
+Because the key is taken from the top-frame (`https://a.example`) and the
+immediate frame which loads the resource (`https://c.example`), a cache hit
+occurs.
+
+<div class="clearfix"></div>
+
+## FAQs
```

Comment:
This section can stay, but I offer a question as food for thought. If you answered these questions in the course of an article, why would anyone need to ask them?

Reply:
That's a fair point, but as this blog post has gone through approval process from PR, legal, marketing etc and took a loooong time, I'm hesitant to make significant changes. And I think the way it is now looks okay.